packages/bundler-webpack/src/config/progressPlugin.ts

```diff
@@ -14,11 +14,13 @@ interface IOpts {
 export async function addProgressPlugin(opts: IOpts) {
   const { config, name, env } = opts;
   // build only since build is too long and no web progress
-  if (env === 'production') {
+  if (env === Env.production) {
     config.plugin('progress-plugin').use(WebpackBar, [
       {
         name: name || 'webpack',
       },
     ]);
+  } else {
+    config.plugin('progress-plugin').use(ProgressPlugin);
```

Comment:
```suggestion
    config.plugin('progress-plugin-dev').use(ProgressPlugin);
```

名字最好别叫一样的，用户修改的时候同一个 name 对应两个不一样的插件，破坏预期。